addon/utils/has-block.js

```diff
@@ -0,0 +1,30 @@
+import Ember from 'ember';
+
+const [major, minor] = Ember.VERSION.split('.');
+const isGlimmer = major > 2 || (major == 2 && minor >= 10); // >= 2.10
+
+let hasBlockSymbol;
+
+if (major == 3 && minor >= 1) {
```

Comment:
Maybe should be `major >=3`, otherwise Ember 4 will try to do the thing that's likely wrong below and load the symbol.

Reply:
Agree. Once we have moved from `ember-browserify` to `ember-auto-import` I plan to bring in `semver` to handle this. For this case I will update it to `major > 3 || (major == 3 && minor >= 1)`